fluent-package/templates/package-scripts/fluent-package/deb/postinst

```diff
@@ -7,25 +7,31 @@ set -e
 prevver="$2"
 
 add_system_user() {
-    if ! getent passwd <%= compat_service_name %> >/dev/null; then
-        adduser --group --system --home /var/lib/<%= package_dir %> <%= compat_service_name %>
+    if ! getent passwd _<%= service_name %> >/dev/null; then
+	if ! getent group <%= compat_service_name %> >/dev/null; then
+	    # With underscore prefix, need to disable NAME_REGEX restriction
+	    adduser --group --system --force-badname --home /var/lib/<%= package_dir %> _<%= service_name %>
+	else
+	    groupmod --new-name _<%= service_name %> <%= compat_service_name %>
+	    usermod --login _<%= service_name %> <%= compat_service_name %>
```

Comment:
Aren't them (user & group) needed to check separately like RPM's one?

Reply:
It should be checked. thanks.